app/api/server/v1/voip/asterisk-connector.ts

```diff
@@ -0,0 +1,87 @@
+
+import { API } from '../../api';
+import { Commands } from '../../../../../server/services/voip/connector/asterisk/Commands';
+import { CommandHandler } from '../../../../../server/services/voip/connector/asterisk/CommandHandler';
+import { IVoipExtensionBase, IVoipExtensionConfig } from '../../../../../definition/IVoipExtension';
+
+
+const commandHandler = new CommandHandler();
+// Get the connector version and type
+API.v1.addRoute('connector.getversion', { authRequired: true }, {
+	get() {
+		return API.v1.success(commandHandler.getVersion());
+	},
+});
+
+// Get the extensions available on the call server
+API.v1.addRoute('connector.extension.list', { authRequired: true }, {
+	get() {
+		try {
+			const list = Promise.await (
+				commandHandler.executeCommand(Commands.extension_list, undefined)) as IVoipExtensionBase;
+			console.log(`JSON=${ JSON.stringify(list) }`);
```

Comment:
```suggestion
this.logger.debug(`JSON=${ JSON.stringify(list) }`);
```

Reply:
the API class has a logger method inbuilt which we can use for logging instead of raw console logs

Reply:
Fixed.